### ISSUE TITLE
gcc 10+ header missing for throwable.h

### ISSUE THREAD

**Issue by otruffer** [CopernicaMarketingSoftware/PHP-CPP#458]
When including phpcpp.h and compiling with gcc 10, you get the following error:

```
In file included from /usr/include/phpcpp.h:38,
                 from /opt/php-optimeyes/src/PhpCorneaDataType.h:4,
                 from /opt/php-optimeyes/src/PhpCorneaDataType.cpp:1:
/usr/include/phpcpp/throwable.h:29:1: error: expected class-name before '{' token
   29 | {
[...]
```
This can be resolved either by requesting the user to include <stdexcept> before including <phpcpp.h>, or by merging https://github.com/CopernicaMarketingSoftware/PHP-CPP/pull/457.

Best
Oskar

**Comment by otruffer** [comment]
My fix seems to be a duplicate of: https://github.com/CopernicaMarketingSoftware/PHP-CPP/pull/440

**Comment by otruffer** [comment]
My fix seems to be a duplicate of: https://github.com/CopernicaMarketingSoftware/PHP-CPP/pull/440

**Comment by robocoder** [comment]
Fixed in PHP-CPP 2.4.0 (c2b36b9843b4b98f67342f6c0a956d1217bcb948)
